sklearn/preprocessing/tests/test_imputation.py

```diff
@@ -364,3 +373,229 @@ def test_imputation_copy():
 
     # Note: If X is sparse and if missing_values=0, then a (dense) copy of X is
     # made, even if copy=False.
+
+
+def test_mice_pipeline_grid_search():
```

Comment:
not sure why we need this if common tests are run.

Reply:
Let me know if I should take it out.